projects/discussion-ui/src/lib/models/discussion-config.model.ts

```diff
@@ -2,7 +2,7 @@ export interface IdiscussionConfig {
     menuOptions?: Array<IMenuOptions>
     userName: string
     context: Object
-    categories: Array<string>
+    categories: Object
```

Comment:
create a separate interface on how individual category will look like. i.e, what properties does that object contain